timesketch/frontend/src/components/Sigma/SigmaList.vue

```diff
@@ -0,0 +1,156 @@
+          <span class="icon is-small">
+                <router-link :to="{ name: 'SigmaContent', query: {ruleId: props.row.id }}"
+                   ><i class="fas fa-search">aaaaaaa</i></router-link></span>
+          <span
+                class="icon is-small"
+                style="cursor:pointer;float:right;"
+                title="Explore"
+                v-on:click="openSigmaRule(props.row.id)">
+          <i class="fas fa-search"></i></span>
+        </b-table-column>
+        <b-table-column field="title" label="Title" v-slot="props" sortable searchable>
+          {{ props.row.title }} <span class="icon is-small"
+                style="cursor:pointer;float:right;"
+                title="Copy value"
+                v-on:click="copyCode(props.row.title)"
+                ><i class="fas fa-copy"></i></span>
+        </b-table-column>
+        <b-table-column field="es_query" label="ES Query" v-slot="props" sortable searchable>
+          {{ props.row.es_query }}
+                <span class="icon is-small">
+                <router-link :to="{ name: 'Explore', query: {q : props.row.es_query} }"
+                   ><i class="fas fa-search"></i></router-link></span>
+                <span class="icon is-small"
+                style="cursor:pointer;float:right;"
+                title="Copy value"
+                v-on:click="copyCode(props.row.es_query)"
+                ><i class="fas fa-copy"></i></span>
+        </b-table-column>
+        <b-table-column field="file_name" label="File Name" v-slot="props" sortable searchable>
+          {{ props.row.file_name }} <span 
+                class="icon is-small" 
+                @mouseover="hover = true"
+                @mouseleave="hover = false"
+                :class="{ active: hover }"
+                style="cursor:pointer;float:right;"
+                title="Copy value"
+                v-on:click="copyCode(props.row.file_name)"
+                ><i class="fas fa-copy"></i></span>
+                <span class="icon is-small">
+                <router-link
+                      :to="{ name: 'Explore', query: {q : props.row.file_name} }"
+                   ><i class="fas fa-search"></i></router-link></span>
+        </b-table-column>
+        <b-table-column field="tags" label="Tag" v-slot="props" sortable searchable>
+          <ul class="content-list">
+            <li style="padding:10px;border-bottom:none;cursor:pointer;">
+              <div v-for="tag in props.row.tags" v-bind:key="tag">
+                {{ tag }} 
+                <span class="icon is-small">
+                <router-link
+                      :to="{ name: 'Explore', query: {q : tag} }"
+                   ><i class="fas fa-search"></i></router-link></span>
+            </div>
+            </li>
+          </ul>
+        </b-table-column>
+      </b-table>
+  </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      sigma_list: [],      
```

Comment:
list of what? Sigma rules? then maybe call it `sigmaRuleList`? WDYT

Reply:
camel case for JS variables

Reply:
got rid of it as it was no longer needed